src/coreclr/jit/compiler.cpp

```diff
@@ -5106,9 +5104,7 @@ void Compiler::compCompile(void** methodCodePtr, uint32_t* methodCodeSize, JitFl
     compJitTelemetry.NotifyEndOfCompilation();
 #endif
 
-#if defined(DEBUG)
-    ++Compiler::jitTotalMethodCompiled;
-#endif // defined(DEBUG)
+    Compiler::jitTotalMethodCompiled++;
```

Comment:
Do you need to make this counter thread-safe now that it is meant for public consumption?

Reply:
Good point! fixed